indico/util/packaging.py

```diff
@@ -7,19 +7,18 @@
 
 import sys
 from importlib.util import find_spec
+from importlib.metadata import distribution
 from pathlib import Path
 
-import pkg_resources
-
 
 def package_is_editable(package):
     """Check whether the Python package is installed in 'editable' mode."""
     # based on pip.dist_is_editable
-    dist = pkg_resources.get_distribution(package)
+    dist = distribution(package)
```

Comment:
In case you want to test the behavior below to make sure `dist.name` is the same: The `egg-link` is created when you install indico in a venv that has setuptools, the `__editable__` is created when setuptools is *not* installed in the venv.

Reply:
works fine